types/spotify-api/index.d.ts

```diff
@@ -263,10 +263,13 @@ declare namespace SpotifyApi {
     }
 
     /**
-     * Get audio analysis for a track
+     * Get Audio Analysis for a Track
      *
      * GET /v1/audio-analysis/{id}
-     * https://developer.spotify.com/documentation/web-api/reference/#/operations/get-audio-analysis
+     * https://developer.spotify.com/web-api/get-audio-analysis/
```

Comment:
It's a bit annoying that some documentation is available using the "short form" (which is much nicer), as this leads to the different documentation links looking non-uniform. Having ones without any anchor hash is much better for the developer experience :/

Reply:
It appears that they only support the redirect from the short form on "old" endpoints. E.g.:
- https://developer.spotify.com/web-api/get-audio-analysis works
- https://developer.spotify.com/web-api/save-shows-user does not

We can't really do anything about it, unfortunately.

There's also https://developer.spotify.com/web-api/object-model which does not exist anymore. Tough.